### ISSUE TITLE
Publish last version of "expo-server-sdk-node" source

### ISSUE THREAD

**Issue by outaTiME** [expo/expo#6167]
Hi guys, it will great to get the last version of "expo-server-sdk-node" source published, especially for the following commit:

https://github.com/expo/expo-server-sdk-node/commit/4658edea542b2c297dab2dbcbf4011db94da8ab7

in v.3.3.0 If i send the same message for multiple recipients the sendPushNotificationsAsync fails, 

For example:

```
const message = {
  to: [
    'ExponentPushToken[123]',
    'ExponentPushToken[456]',
  ],
  body: 'Foo message',
  sound: 'default',
};
```

Error:

```
Error: Expected Expo to respond with 1 ticket but got 2
    at Expo.<anonymous> (/private/var/folders/vm/z64z40tn2jj9nqc7znvxvk240000gn/T/zeit-fun-840b842faf7de/node_modules/expo-server-sdk/build/ExpoClient.js:76:32)
    at Generator.next (<anonymous>)
    at fulfilled (/private/var/folders/vm/z64z40tn2jj9nqc7znvxvk240000gn/T/zeit-fun-840b842faf7de/node_modules/expo-server-sdk/build/ExpoClient.js:4:58)
    at processTicksAndRejections (internal/process/task_queues.js:93:5) {
  data: [
    { id: '1bd95e36-8368-4d40-a64a-b3efa9917212', status: 'ok' },
    { status: 'ok', id: '48ae888f-8541-4463-bce7-9ae9dccec550' }
  ]
}
```

**Comment by ide** [comment]
Published 3.4.0.

**Comment by outaTiME** [comment]
Thanks @ide, it works as expected 👍
